features/step_definitions/environments.yml

```diff
@@ -5,6 +5,9 @@ local:
     Stub Country: http://localhost:50140/eidas/stub-country/login
     Stub Idp Demo One: http://localhost:50140/stub-idp-demo-one/login
     Stub Idp Demo Two: http://localhost:50140/stub-idp-demo-two/login
+  idp-resume-links:
```

Comment:
can we remove these from the environments file and move them to the steps... given these properties are not environment-specific (they are all the same) they don't need to be here. Similar what you did in `features/step_definitions/user_steps.rb` line 497.